@here/harp-mapview/lib/MapView.ts

```diff
@@ -667,6 +667,12 @@ export interface MapViewOptions extends TextElementsRendererOptions, Partial<Loo
      * If set, the view will constrained within the given bounds in geo coordinates.
      */
     maxBounds?: GeoBox;
+
+    /**
+     * Optional [[ElevationProvider]] where [[DataSource]]s create geometry above and/or below
```

Comment:
Optional `ElevationRangeSource`.
Initially the `ElevationRangeSource` was intended to only be used in combination with terrain (i.e. `ElevationProvider`). If the datasource has data that is not flat it would provide it's range in min/maxGeometry height (constant) or per tile.
When terrain is enabled and a ElevationRangeSource is set, the final tile bounding box would be the accumulation of terrain min/max height + datasource min/ max height.

As far as I understood the issue, it is  only about the min/max elevation range in the datasource.

Reply:
I disagree: The ElevationRangeSource is required for visibility testing during loading. Without it, probable tiles will not be loaded. The min/max elevation range is used for bounding box calculation during rendering. The ElevationRangeSource is required for this use case.

Reply:
Both, min/max height and elevation range source, should be used during loading.
The difference is that min/max height is global and the elevation range source is on tile level.

Additionally as mentioned was ElevationRangeSource intended for terrain.
Imagine putting vector data on top of terrain. To get propper bounding boxes we would need two elevation range sources, one for terrain and one for the vector data.
To avoid this complexity of accumulating multiple ElevationRangeSources the idea was that there is only one ElevationRangeSource for terrain and the min/max height for the vector data. The final bounding box of a tile would be the elevation range + min/max height.